caffe2/opt/optimize_ideep.cc

```diff
@@ -37,15 +37,23 @@ T* getTensor(Blob* blob) {
   return nullptr;
 }
 
-const caffe2::OperatorDef& getOpDef(const repr::NeuralNetOperator& nnOp) {
+template <class T> T *getMutableTensor(Blob *blob) {
+  CAFFE_ENFORCE(blob, "Blob is invalid");
+  if (blob && blob->template IsType<T>()) {
+    return blob->template GetMutable<T>();
+  }
+  return nullptr;
+}
+
+const caffe2::OperatorDef &getOpDef(const repr::NeuralNetOperator &nnOp) {
   auto annotation = nnOp.getAnnotation();
   if (annotation == nullptr) {
     CAFFE_THROW("Cannot get Operator annotation");
   }
   return dyn_cast<Caffe2Annotation>(annotation)->getOperatorDef();
 }
 
-caffe2::OperatorDef* getMutableOpDef(repr::NeuralNetOperator& nnOp) {
+caffe2::OperatorDef *getMutableOpDef(repr::NeuralNetOperator &nnOp) {
```

Comment:
Could you align your clang format with the one used in PyTorch? This doesn't seem right.